### PR TITLE
[FIX] Download page alignment

### LINKED ISSUE #70: Download page alignment
Text on the "Download" page should be left-aligned with the Orange logo in the top menu. Now it is not. Depending on the size of the browser the text is either too wide (on very wide windows), too narrow, or, for a specific size, the logo magically jumps inward.

The same goes for all pages, but let's start with Download. This will likely also fix some problems with other pages.

The magical inwards logo jump:
test

### PR DESCRIPTION
**Pull request by mitjapotocin**
-Content is now left-aligned with navigation bar logo. 

-Fixes navigation bar logo magical inward jumping, when screen size is changed. 

-OS icons are now vertically aligned on small screen sizes. 

Fixes #70

### CODE REVIEW

**Comment by mitjapotocin** [comment]
Content is now left-aligned with navigation bar logo. 
Fixes avigation bar logo magical inward jumping on window resize. 

Other pages might need some TLC allso.

**Comment by markotoplak** [comment]
@mitjapotocin, thanks, everything seems fine.

Why did you decide to put the icons vertically? If you do that, they take a lot of vertical space. On my phone, they still fit horizontally. I would avoid vertical composition here. If you think they are too big, perhaps we could make them (and text) smaller on xs-sized screens.

File: static/css/styles.css [hunk 1]
```diff
@@ -951,9 +953,9 @@ svg{
   /*.container{
     padding: 0px !important;
   }*/
-  .header-pad{
+  /* .header-pad{
```
Comment (markotoplak) [inline static/css/styles.css]: You could delete this.

File: layouts/download/list.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {{ define "main"}}
 
 <section id="features" class="features section downloads ">
-        <div class="container-fluid downloados downloadsize" style="margin-top: 50px;">
+        <div class="container downloados" style="margin-top: 50px;">
             <div class="row" style="padding: 0px !important;text-align: center;">
                 <div class="dow" style="display: inline-block; width: 80%;">
                     {{ range .Site.Data.tabs}}
-                    <div class="icon-item col-xl-4 col-lg-4 col-md-4 col-sm-4  col-xs-4 text-center" >
+                    <div class="icon-item col-xl-4 col-lg-4 col-md-4 col-sm-4  col-xs-14 text-center" >
```
Comment (markotoplak) [inline layouts/download/list.html]: I know the original code was not beautiful, but please do not reference nonexistent classes.

**Comment by mitjapotocin** [comment]
On my phone the icons were too close together, that is why I decided, to align them vertically, like on the old website. 
I agree with you, it is better to align them horizontally. 
I aligned them horizontally again and removed the padding on small screens, so the icons can fit.

**Comment by markotoplak** [comment]
Thank you! Works great.

**Comment by markotoplak** [comment]
@mitjapotocin, now that you dug into the code: do you have any suggestions of what should be improved next?

**Comment by mitjapotocin** [comment]
I think all the pages should have this look (left-aligned with nav logo).
I noticed, that Blog page has a completely different style. Is that intentional?

**Comment by markotoplak** [comment]
@mitjapotocin, I do not know if it was intentional, but I also think it should be unified with the rest (opened #81).

**Pull request merged**